### PR TITLE
Added Token assets table

### LINKED ISSUE #121: UI: Create table component
Screen Shot 2022-01-28 at 5 46 33 pm

Needs to work in other places e.g. connections page

### LINKED ISSUE #122: UI: Create tabbing component
Might be useful: https://github.com/jzaki/bls-wallet/blob/main/extension/source/Popup/components/Tabs.tsx

### LINKED ISSUE #127: UI: Create wallets->assets tab without filter
Depends on #119, #122, #121

### PR DESCRIPTION
**Pull request by kautukkundan**
## What is this PR doing?
Adds wallet detail on right side of wallets page
Screenshot 2022-02-22 at 1 11 23 AM

## How can these changes be manually tested?
build and import extension

## Does this PR resolve or contribute to any issues?
closes #121
closes #122
closes #127 

## TODO
https://github.com/jzaki/bls-wallet/issues/144
https://github.com/jzaki/bls-wallet/issues/145
https://github.com/jzaki/bls-wallet/issues/146
https://github.com/jzaki/bls-wallet/issues/148

## Checklist
- [x] I have manually tested these changes
- [x] Post a link to the PR in the group chat

## Guidelines
- If your PR is not ready, mark it as a draft
- The `resolve conversation` button is for reviewers, not authors
  - (But add a 'done' comment or similar)

### CODE REVIEW

File: extension/source/QuillPage/Wallet/Wallets/AssetsTable.tsx [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+      </div>
+    </div>
+  );
+};
+
+interface IAssetsTable {
+  data: TokenData[];
+}
+
+export const AssetsTable: React.FunctionComponent<IAssetsTable> = ({
```
Comment (kautukkundan) [inline extension/source/QuillPage/Wallet/Wallets/AssetsTable.tsx:47]: Need to make this generic so single table can support Assets and Transactions 
tracking in #146

File: extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+export interface TokenData {
+  token: string;
+  tokenVal: number;
+  usdVal: number;
+  lastTx: string;
+  action: string;
+}
+
+const data: TokenData[] = [
```
Comment (kautukkundan) [inline extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx:12]: this data will ideally come from a controller, using a mockup here

File: extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx [hunk 3]
```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import { AssetsTable } from './AssetsTable';
+
+export interface TokenData {
+  token: string;
+  tokenVal: number;
+  usdVal: number;
+  lastTx: string;
```
Comment (kautukkundan) [inline extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx:8]: will use a date type once the data coming from controller is finalized

File: extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+
+  return (
+    <div className="flex border-b border-grey-300 gap-4 mb-4">
+      {tabs.map((tab) => (
+        <div
+          key={tab.name}
+          className={`py-2 px-4 cursor-pointer ${
+            tab.name === activeTab && 'border-b-2 border-blue-500 text-blue-500'
+          }`}
+          onClick={() => setActiveTab(tab.name)}
```
Comment (kautukkundan) [inline extension/source/QuillPage/Wallet/Wallets/WalletDetail.tsx:70]: setting active tab currently doesn't do anything, just changes the active state. tracking in
#144

**Comment by voltrevo** [comment]
Nicely done 👍

**Pull request merged**